src/java/picard/analysis/AdapterUtil.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2015 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package picard.analysis;
+
+import htsjdk.samtools.util.CollectionUtil;
+import htsjdk.samtools.util.SequenceUtil;
+import htsjdk.samtools.util.StringUtil;
+import picard.util.IlluminaUtil;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class AdapterUtil {
+
+    //The number of bases to check in order to map a read to an adapter
+    private static final int ADAPTER_MATCH_LENGTH = 16;
+
+    // The maximum number of mismatches a read can have and still be considered as matching an adapter
+    private static final int MAX_ADAPTER_ERRORS = 1;
+
+    public static List<String> DEFAULT_ADAPTER_SEQUENCE = CollectionUtil.makeList(
+            IlluminaUtil.IlluminaAdapterPair.SINGLE_END.get5PrimeAdapter(),
+            IlluminaUtil.IlluminaAdapterPair.SINGLE_END.get3PrimeAdapter(),
+            IlluminaUtil.IlluminaAdapterPair.PAIRED_END.get5PrimeAdapter(),
+            IlluminaUtil.IlluminaAdapterPair.PAIRED_END.get3PrimeAdapter(),
+            IlluminaUtil.IlluminaAdapterPair.INDEXED.get5PrimeAdapter(),
+            IlluminaUtil.IlluminaAdapterPair.INDEXED.get3PrimeAdapter()
+    );
```

Comment:
Just spoke to @ktibbett.  I'm not going to add that into this PR because it could change downstream results, but am adding a TODO in the code and will file an issue in github.